src/utils/make-command.js

```diff
@@ -1,7 +1,34 @@
 const { EOL } = require( 'os' );
+const chalk = require( 'chalk' );
+const logSymbols = require( 'log-symbols' );
+const makeDocker = require( '../utils/make-docker' );
 
-module.exports = function makeCommand( chalk, { error }, command ) {
+/**
+ * Check if Docker app is running.
+ *
+ * @return {Promise<boolean|*>}
+ */
+async function pingDocker() {
+    try{
+        const docker = makeDocker();
+        return await docker.ping();
+    } catch( ex ) {
+        return false;
+    }
+}
+
+module.exports = function makeCommand( options = {}, { error }, command ) {
+    const { checkDocker = true } = options;
     return ( ...params ) => {
+        // Check if Docker is running.
+        if ( checkDocker === true ) {
+            pingDocker().then( result => {
+                if ( result === false ) {
+                    console.log( logSymbols.error, chalk.red( 'Docker is not running...' ) );
```

Comment:
Let's extract error rendering into a separate function to use it here and in the command hanler catch.